### PR TITLE
integrated maps coordinations by business submit form

### PR DESCRIPTION
**Pull request by Davoshy**

### CODE REVIEW

File: frontend/src/views/NewStore/BusinessForm.js [hunk 1]
```diff
@@ -57,8 +57,16 @@ const BusinessForm = ({ location, history }) => {
 
   const submitHandler = async (e) => {
     e.preventDefault();
+    let lat = 0;
+    let lon = 0;
+    let searchString = data.line1 + " , " + data.line2;
+    const getLocation = await axios(
+        `https://eu1.locationiq.com/v1/search.php?key=${process.env.REACT_APP_KEY}&q=${searchString}&format=json`)
+    lat = Number(getLocation.data[0].lat).toFixed(5);
+    lon = Number(getLocation.data[0].lon).toFixed(5)
```
Comment (vellip) [inline frontend/src/views/NewStore/BusinessForm.js:66]:
```suggestion
    const latitude = Number(getLocation.data[0].lat).toFixed(5);
    const longitude = Number(getLocation.data[0].lon).toFixed(5)
```

File: frontend/src/views/NewStore/BusinessForm.js [hunk 2]
```diff
@@ -57,8 +57,16 @@ const BusinessForm = ({ location, history }) => {
 
   const submitHandler = async (e) => {
     e.preventDefault();
+    let lat = 0;
+    let lon = 0;
```
Comment (vellip) [inline frontend/src/views/NewStore/BusinessForm.js:61]:
```suggestion
```

File: frontend/src/views/NewStore/BusinessForm.js [hunk 3]
```diff
@@ -57,8 +57,16 @@ const BusinessForm = ({ location, history }) => {
...
+    let searchString = data.line1 + " , " + data.line2;
+    const getLocation = await axios(
+        `https://eu1.locationiq.com/v1/search.php?key=${process.env.REACT_APP_KEY}&q=${searchString}&format=json`)
+    lat = Number(getLocation.data[0].lat).toFixed(5);
+    lon = Number(getLocation.data[0].lon).toFixed(5)
+
     const response = await API.partners.post({
-      ...data, address: { ...data }, users: [user.id]
+      ...data, address: { ...data, latitude: lat, longitude: lon }, users: [user.id]
```
Comment (vellip) [inline frontend/src/views/NewStore/BusinessForm.js:69]:
```suggestion
      ...data, address: { ...data, latitude, longitude }, users: [user.id]
```

File: frontend/src/views/NewStore/BusinessForm.js [hunk 4]
```diff
@@ -6,7 +6,7 @@ import { useTranslation } from 'react-i18next';
 import React, { useState } from 'react';
 import { useUserContext } from '../../context/UserContext';
 import API from '../../shared/api';
-
+import axios from "axios"
```
Comment (vellip) [inline frontend/src/views/NewStore/BusinessForm.js:9]: single quotes please

**Comment by Davoshy** [comment]
REACT_APP_KEY = pk.4c61e48b53acaa5cd9ae20ab6f019f18

**Pull request merged**